contributors/guide/release-notes.md

```diff
@@ -10,7 +10,7 @@ output format changes. Release notes are one of the most important reference
 points for users about to install or upgrade to a particular release of
 Kubernetes.
 
-## Does my pull request need a release note?
+## Does my pull request needs a release note?
```

Comment:
```suggestion
## Does my pull request need a release note?
```

Reply:
I'm a fan of those suggested changes in GitHub (hopefully makes it easier to update on the go!)

Reply:
Thanks @onlydole :pray: